tests/gold_tests/redirect/redirect.test.py

```diff
@@ -108,4 +108,43 @@
 tr.Processes.Default.Streams.stdout = "gold/redirect.gold"
 tr.Processes.Default.ReturnCode = 0
 
+
+for status,phrase in sorted({
+        301:'Moved Permanently',
+        302:'Found',
+        303:'See Other',
+        305:'Use Proxy',
+        307:'Temporary Redirect',
+        308:'Permanent Redirect',
+        }.items()):
```

Comment:
Note to reviewers: I found we were not already testing the other status codes for which we redirect, so I went ahead and added test coverage for all of them.